src/demo/toMode.js

```diff
@@ -0,0 +1,9 @@
+import {init as initModel} from './models';
```

Comment:
I had a 5 file circular dependency, so I moved this method out. Definitely open to other thoughts about where we should put this or how we should split up the `helpers` files.